.storybook/preview-head.html

```diff
@@ -0,0 +1,5 @@
+<style>
+  #root {
+    height: 100%;
+  }
+</style>
```

Comment:
Setting the height of the components preview container to an explicit value. This is useful for components that rely to a parent element with an explicit height.